<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataAnalyzerClass</class>
 <widget class="QMainWindow" name="DataAnalyzerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1295</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Analysis</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resource/DataAnalyzer.qrc">
    <normaloff>:/images/1467665209-300px.png</normaloff>:/images/1467665209-300px.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="iconSize">
   <size>
    <width>36</width>
    <height>36</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Directory</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTreeView" name="detailedDataItemTreeView">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectItems</enum>
            </property>
            <property name="rootIsDecorated">
             <bool>false</bool>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="animated">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Analysis</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="30,8">
          <item>
           <widget class="QPlainTextEdit" name="analysisTextEdit">
            <property name="placeholderText">
             <string>..... own analysis .....</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_5">
            <property name="title">
             <string>Tags</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QListWidget" name="tagListWidget">
               <item>
                <property name="text">
                 <string>No tag</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Not interesting</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Interesting</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Proof</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>Preview</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>283</width>
              <height>353</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="previewWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="textFormat">
                <enum>Qt::PlainText</enum>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="textInteractionFlags">
                <set>Qt::NoTextInteraction</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="counterClockwiseButton">
             <property name="text">
              <string>rotate -90°</string>
             </property>
             <property name="icon">
              <iconset resource="../../resource/DataAnalyzer.qrc">
               <normaloff>:/images/primary-rotate-ccw-800px.png</normaloff>:/images/primary-rotate-ccw-800px.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clockwiseButton">
             <property name="text">
              <string>rotate +90°</string>
             </property>
             <property name="icon">
              <iconset resource="../../resource/DataAnalyzer.qrc">
               <normaloff>:/images/primary-rotate-cw-800px.png</normaloff>:/images/primary-rotate-cw-800px.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Detailed information</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTreeWidget" name="detailsTreeWidget">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="itemsExpandable">
           <bool>false</bool>
          </property>
          <property name="animated">
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <property name="expandsOnDoubleClick">
           <bool>false</bool>
          </property>
          <property name="columnCount">
           <number>2</number>
          </property>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Detail</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1295</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSoubor">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuN_pov_da">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="action3rdParty"/>
   </widget>
   <widget class="QMenu" name="menu_pravy">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuNastaven">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuOperace">
    <property name="title">
     <string>Operation</string>
    </property>
    <widget class="QMenu" name="menuGen">
     <property name="title">
      <string>Final reports</string>
     </property>
     <addaction name="actionFinalReportCSV"/>
     <addaction name="actionFinalReportRTF"/>
    </widget>
    <addaction name="actionScan"/>
    <addaction name="actionMD5"/>
    <addaction name="actionFileMagic"/>
    <addaction name="separator"/>
    <addaction name="menuGen"/>
   </widget>
   <addaction name="menuSoubor"/>
   <addaction name="menu_pravy"/>
   <addaction name="menuOperace"/>
   <addaction name="menuNastaven"/>
   <addaction name="menuN_pov_da"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Other operations</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Main operations</string>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionScan"/>
   <addaction name="actionMD5"/>
   <addaction name="actionFileMagic"/>
   <addaction name="separator"/>
   <addaction name="actionFinalReportCSV"/>
   <addaction name="actionFinalReportRTF"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>File operations</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>Directories</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QCheckBox" name="dirOnlyCheckBox">
           <property name="text">
            <string>Directories only</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeView" name="dataItemTreeView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="uniformRowHeights">
            <bool>true</bool>
           </property>
           <property name="animated">
            <bool>true</bool>
           </property>
           <attribute name="headerCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Search</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Tags</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../resource/DataAnalyzer.qrc">
     <normaloff>:/images/Exit-by-Rones-800px.png</normaloff>:/images/Exit-by-Rones-800px.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string notr="true">Meta+Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="action3rdParty">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>About components ...</string>
   </property>
   <property name="toolTip">
    <string>About 3rd parties components</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="icon">
    <iconset resource="../../resource/DataAnalyzer.qrc">
     <normaloff>:/images/frame-radar-icon-001.png</normaloff>:/images/frame-radar-icon-001.png</iconset>
   </property>
   <property name="text">
    <string>Directory scan ...</string>
   </property>
   <property name="toolTip">
    <string>Initial file system scan</string>
   </property>
   <property name="shortcut">
    <string notr="true">Meta+Ctrl+S</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Cut</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+V</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Open ...</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+O</string>
   </property>
  </action>
  <action name="actionFinalReportCSV">
   <property name="icon">
    <iconset resource="../../resource/DataAnalyzer.qrc">
     <normaloff>:/images/frame-adriankierman-report-page-csv.png</normaloff>:/images/frame-adriankierman-report-page-csv.png</iconset>
   </property>
   <property name="text">
    <string>CSV</string>
   </property>
  </action>
  <action name="actionMD5">
   <property name="icon">
    <iconset resource="../../resource/DataAnalyzer.qrc">
     <normaloff>:/images/frame-fingerprint.png</normaloff>:/images/frame-fingerprint.png</iconset>
   </property>
   <property name="text">
    <string>MD5 computation ...</string>
   </property>
   <property name="shortcut">
    <string notr="true">Meta+Ctrl+M</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Search ...</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+F</string>
   </property>
  </action>
  <action name="actionFileMagic">
   <property name="icon">
    <iconset resource="../../resource/DataAnalyzer.qrc">
     <normaloff>:/images/frame-fingerprint-magic.png</normaloff>:/images/frame-fingerprint-magic.png</iconset>
   </property>
   <property name="text">
    <string>Data type analysis</string>
   </property>
   <property name="shortcut">
    <string notr="true">Meta+Ctrl+T</string>
   </property>
  </action>
  <action name="actionFinalReportRTF">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../resource/DataAnalyzer.qrc">
     <normaloff>:/images/frame-adriankierman-report-page-rtf.png</normaloff>:/images/frame-adriankierman-report-page-rtf.png</iconset>
   </property>
   <property name="text">
    <string>RTF</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resource/DataAnalyzer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScan</sender>
   <signal>triggered()</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>onScanTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>analysisTextEdit</sender>
   <signal>textChanged()</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>onAnalysisTextChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>detailedDataItemTreeView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>onDetailedDataItemSelected(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>553</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dataItemTreeView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>onDataItemSelected(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dirOnlyCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>onDirOnly(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clockwiseButton</sender>
   <signal>clicked()</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>onClockwiseRotate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1168</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>counterClockwiseButton</sender>
   <signal>clicked()</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>onCounterClockwiseRotate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1087</x>
     <y>456</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tagListWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>onTagChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>888</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>onAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMD5</sender>
   <signal>triggered()</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>onMD5Triggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFileMagic</sender>
   <signal>triggered()</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>onFileMagicTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFinalReportRTF</sender>
   <signal>triggered()</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>onRTFReportTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFinalReportCSV</sender>
   <signal>triggered()</signal>
   <receiver>DataAnalyzerClass</receiver>
   <slot>onCSVReportTriggered()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onScanTriggered()</slot>
  <slot>onDataItemSelected(QModelIndex)</slot>
  <slot>onAnalysisTextChange()</slot>
  <slot>onDetailedDataItemSelected(QModelIndex)</slot>
  <slot>onDirOnly(bool)</slot>
  <slot>onCounterClockwiseRotate()</slot>
  <slot>onClockwiseRotate()</slot>
  <slot>onTagChanged(int)</slot>
  <slot>onAbout()</slot>
  <slot>onMD5Triggered()</slot>
  <slot>onFileMagicTriggered()</slot>
  <slot>onCSVReportTriggered()</slot>
  <slot>onRTFReportTriggered()</slot>
 </slots>
</ui>
